### PR TITLE
Improve main SDK page

### PR DESCRIPTION
**Pull request by feorlen**
Small cleanup of the main SDK index page. 

[x] More clearly link to respective Quickstarts, API references, and GitHub repos.
[x] Reorder more-or-less by popularity.

Non-goals:
Validating the install steps provided (except replace `go get` with `go install ...@version`)

Staged:
http://192.241.195.202:9000/staging/main-sdk-page/linux/developers/minio-drivers.html

### CODE REVIEW

File: source/developers/minio-drivers.rst [hunk 1]
```diff
@@ -12,113 +12,75 @@ Software Development Kits (SDK)
...
 
-     PM> Install-Package Minio
+GitHub: `minio/minio-py <https://github.com/minio/minio-py>`__
 
-.. _go-sdk:
+Latest Version: |python-sdk-version|
 
-Go (``minio-go``)
------------------
+Quickstart Guide: :doc:`/developers/python/minio-py`
```
Comment (ravindk89) [inline source/developers/minio-drivers.rst:50]: We do also have the reference. We could point to both here consistently.

File: source/developers/minio-drivers.rst [hunk 2]
```diff
@@ -12,113 +12,75 @@ Software Development Kits (SDK)
...
-.. _java-sdk:
+GitHub: `minio/minio-java <https://github.com/minio/minio-java>`__
 
-Java (``minio-java``)
----------------------
+Latest version: |java-sdk-version|
 
-Latest version
-  |java-sdk-version|
+Reference: :ref:`minio-java-quickstart`
```
Comment (ravindk89) [inline source/developers/minio-drivers.rst:79]: This is the quickstart, but we can also point out the reference.

**Review by ravindk89 — APPROVED** [review]
One nit - for those where we have both reference and quickstart, we might as well link to both. especially since the reference is nested under the top-level sdk landing page/quickstart.

Otherwise LGTM.

File: source/developers/minio-drivers.rst (hunk 1 above)
Comment (feorlen) [reply source/developers/minio-drivers.rst:50]: oo yes

**Review by djwfyi — APPROVED** [review]
Go forth, apply Ravind's feedback, and merge

**Pull request merged**